tests/api/route-services/experiment.test.js

```diff
@@ -8,73 +8,63 @@ describe('tests for the experiment service', () => {
     AWSMock.restore('DynamoDB');
   });
 
-  it('Get experiment data works', async (done) => {
-    const unmarshalledData = {
-      Item: {
-        experimentId: { S: '12345' },
-        experimentName: { S: 'TGFB1 experiment' },
-      },
-    };
-
-    const marshalledData = {
-      experimentId: '12345',
-      experimentName: 'TGFB1 experiment',
+  const mockDynamoGetItem = (jsData) => {
```

Comment:
Extracted mocking from individual test methods, plus took advantage of `AWS.DynamoDB.Converter.marshall` to create mocked data directly from regular js objects